docs/sources/developers/plugins/build-a-streaming-data-source-plugin.md

```diff
@@ -6,27 +6,35 @@ title: Build a streaming data source plugin
 
 This guide explains how to build a streaming data source plugin.
 
+In Grafana, you can set your dashboards to automatically refresh at a certain interval, no matter what data source you use. Unfortunately, this means that your queries are requesting all the data to be sent again, regardless of whether the data has actually changed. The solution to this issue is to add _streaming_ to the plugin.
+
+## Before you begin
+
 This guide assumes that you're already familiar with how to [Build a data source plugin](/tutorials/build-a-data-source-plugin/).
 
-When monitoring critical applications, you want your dashboard to refresh as soon as your data does. In Grafana, you can set your dashboards to automatically refresh at a certain interval, no matter what data source you use. Unfortunately, this means that your queries are requesting all the data to be sent again, regardless of whether the data has actually changed.
+Grafana uses [RxJS](https://rxjs.dev/) to continuously send data from a data source to a panel visualization. There's a lot more to RxJS than what's covered in this guide. 
```

Comment:
Since the note below points to the RxJS documentation, it's redundant to say that there's more than what's covered in this guide since that is shown by the doc link. 
```suggestion
Grafana uses [RxJS](https://rxjs.dev/) to continuously send data from a data source to a panel visualization. 
```

Reply:
Fixed, will check in soon

Reply:
Checked in fix, available for review.